__tests__/app.test.js

```diff
@@ -157,6 +157,28 @@ describe('POST `/api/articles/:article_id/comments` tests', () => {
         });
     })
 })
+describe('PATCH `/api/articles/:article_id', () => {
+    test('200:returns status code 200 upon successful patch request', () => {
+        return request(app)
+        .patch('/api/articles/1')
+        .send({inc_votes:999})
+        .expect(200);
+    })
+    test('200:returns the updated article upon successful patch request', () => {
+        return request(app)
+        .patch('/api/articles/1')
+        .send({inc_votes:999})
+        .then(({body: {updatedArticle}}) => {
+            expect(updatedArticle).toHaveProperty('author', 'butter_bridge');
+            expect(updatedArticle).toHaveProperty('title', "Living in the shadow of a great man");
+            expect(updatedArticle).toHaveProperty('body', "I find this existence challenging");
+            expect(updatedArticle).toHaveProperty('topic', "mitch");
+            expect(updatedArticle).toHaveProperty('created_at', '2020-07-09T20:11:00.000Z');
+            expect(updatedArticle).toHaveProperty('votes', 1099);
+            expect(updatedArticle).toHaveProperty('article_img_url', "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
+        })
+    })
+})
```

Comment:
This is looking good, you can also use toEqual here and make it more DRY as we know what the actual object we expect to receive looks like.